Client/src/Pages/Monitors/CreateMonitor/index.jsx

```diff
@@ -219,33 +212,27 @@ const CreateMonitor = () => {
                 desc="Use HTTP(s) to monitor your website or API endpoint."
                 size="small"
                 value="http"
-                checked={checks.type === "http" || checks.type === "https"}
-                onChange={(event) => handleChange(event, "type", setChecks)}
+                checked={monitor.type === "http"}
+                onChange={(event) => handleChange(event)}
               />
-              {checks.type === "http" || checks.type === "https" ? (
+              {monitor.type === "http" ? (
                 <ButtonGroup sx={{ ml: "32px" }}>
                   <Button
                     level="secondary"
                     label="HTTP"
```

Comment:
Can we change the order of these?  HTTPS should be the most common so it makes more sense to have that as the leftmost and default value I think